apps/api/src/service/email/layouts/templates/EmailRejection.ts

```diff
@@ -36,7 +36,7 @@ export const GenerateEmailRejection = ({ id, name, comment }: Omit<GenerateRejec
                     Thank you for submitting <a href="${ui}/application/${id}" target="_blank" rel="nofollow">your application</a> to the PCGL DACO. After careful review, we regret to inform you that your application has not been approved. As a result, you will not have access to the requested data.
                 </mj-text>
                 <mj-text>
-                    This is the Data Access Committee's comments on your application: ${comment}.
+                    This is the Data Access Committee's comments on your application: ${comment}
```

Comment:
Shouldn't this be plural? There are multiple comments:

```suggestion
                    These are the Data Access Committee's comments on your application: ${comment}
```